scripts/config.py

```diff
@@ -200,6 +200,7 @@ def realfull_adapter(_name, active, section):
     'MBEDTLS_RSA_NO_CRT', # influences the use of RSA in X.509 and TLS
     'MBEDTLS_TEST_CONSTANT_FLOW_MEMSAN', # build dependency (clang+memsan)
     'MBEDTLS_TEST_CONSTANT_FLOW_VALGRIND', # build dependency (valgrind headers)
+    'MBEDTLS_TRACE', # tracing not required for full build
```

Comment:
Given the extreme verbosity of the traces, this should indeed not be enabled in `full`. But then we need a component in `all.sh` that enables this option and at least runs the unit tests (with stderr redirected to a file). This component should also run `process_trace.py` on the output.

Reply:
I'm not sure how to resolve this.